src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -1,5 +1,6 @@
-import { api, track, LightningElement } from 'lwc';
+import {api, track, LightningElement, wire} from 'lwc';
```

Comment:
`wire` and `getObjectInfo` is not used anymore in the file.

Reply:
Good catch.